src/util/config.cpp

```diff
@@ -765,16 +765,38 @@ void configt::set_arch(const irep_idt &arch)
   }
 }
 
+/// \brief Parses the `object_bits` argument from the command line arguments.
+/// \param argument The command line argument to parse the `object_bits` from.
+/// \param pointer_width The width of a pointer in bits. This is used to check
+///   the value of object_bits is within the valid range.
+/// \return A `bv_encodingt` on successful parsing. In the case where an invalid
+///    argument is specified, an `invalid_command_line_argument_exceptiont` will
+///    be thrown.
+configt::bv_encodingt parse_object_bits_encoding(
+  const std::string &argument,
+  const std::size_t pointer_width)
+{
+  const auto object_bits = string2optional<unsigned int>(argument);
+  if(!object_bits || *object_bits <= 0 || *object_bits >= pointer_width)
```

Comment:
Note `<= 0` on an unsigned type

Reply:
I have swapped this to use `== 0`, as this should be less misleading.